app/code/community/Bolt/Boltpay/controllers/ShippingController.php

```diff
@@ -406,26 +344,26 @@ public function getPrefetchCacheIdentifier($quote, $addressData)
     }
 
     /**
-     * @return mixed
+     * Returns a whether P.O. box addresses are allowed for this store
+     *
+     * @return bool     true if P.O. boxes are allowed.  Otherwise, false.
      */
     protected function isPOBoxAllowed()
     {
-        return Mage::getStoreConfig('payment/boltpay/allow_po_box');
+        return (bool) Mage::getStoreConfig('payment/boltpay/allow_po_box');
```

Comment:
I believe you can use `getStoreConfigFlag` instead?

Reply:
Thank you!